docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+# Configuration Variables
+
+This document provides a comprehensive list and explanation of the configuration variables available in the `invenio-app-ils` backend. These variables can be set in the configuration file(s) to customize the behavior of your Invenio ILS application.
+
+## REST Configurations
+
+### `ILS_RECORDS_REST_ENDPOINTS`
```

Comment:
I wouldn't add this one, I think it is too advanced, maybe we need some "advanced"  config section or remove completely